rustls/src/quic.rs

```diff
@@ -701,6 +701,23 @@ pub trait PacketKey: Send + Sync {
         payload: &mut [u8],
     ) -> Result<Tag, Error>;
 
+    /// Encrypts a multipath QUIC packet
+    ///
+    /// Takes a `path_id` and `packet_number`, used to derive the nonce; the packet `header`, which is used as
+    /// the additional authenticated data; and the `payload`. The authentication tag is returned if
+    /// encryption succeeds.
+    ///
+    /// Fails if and only if the payload is longer than allowed by the cipher suite's AEAD algorithm.
+    ///
+    /// See <https://www.ietf.org/archive/id/draft-ietf-quic-multipath-11.html#name-nonce-calculation>.
+    fn encrypt_in_place_multipath(
```

Comment:
I don't think we want to make a semver-incompatible release for this, so if this trait needs new methods those need to have a default implementation.

Reply:
I'm not sure what would be a good default implementation. Silently ignoring the path_id and using the non-multipath method will not produce the right data. Is there a better option to returning an error? The appropriate ones would then be `Error::EncryptError` and `Error::DecryptError`, is that correct?

Reply:
ok this is what I did, let me know if there's a better option

Reply:
I think that's probably a reasonable solution in this case.